src/fidesops/ops/task/graph_task.py

```diff
@@ -88,6 +92,12 @@ def result(*args: Any, **kwargs: Any) -> List[Optional[Row]]:
                     self.log_paused(action_type, ex)
                     # Re-raise to stop privacy request execution on pause.
                     raise
+                except PrivacyRequestErasureEmailSendRequired as exc:
+                    self.log_end(action_type, ex=None, success_override_msg=exc)
+                    self.resources.cache_erasure(
+                        f"{self.traversal_node.address.value}", 0
+                    )  # Cache that the erasure was performed in case we need to restart
+                    return 0
```

Comment:
Following a pattern we use elsewhere - I raise an Exception in the `EmailConnector.mask_data` to update the execution log as it is a concern of the GraphTask and not the individual EmailConnectors.  In this case, I don't want to add a more specific message to the erasure ExecutionLog as the action is not technically completed until we send the email at the end.